dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/DLCAttestationType.scala

```diff
@@ -0,0 +1,29 @@
+package org.bitcoins.dlc.oracle
+
+import org.bitcoins.crypto.CryptoUtil
+import scodec.bits.ByteVector
+
+sealed trait DLCAttestationType {
+  def bytes: ByteVector
+}
+
+case class EnumAttestation(outcome: String) extends DLCAttestationType {
+  def bytes: ByteVector = CryptoUtil.serializeForHash(outcome)
+}
+
+case class RangeAttestation(outcome: Long) extends DLCAttestationType {
```

Comment:
Maybe there can be a shared parent trait called `NumberAttestation` between `RangeAttestation` and `LargeRangeAttestation`?

Reply:
I don't really see the benefit in adding this as there should never be a case where we don't care if it is a `RangeAttestation` or a `LargeRangeAttestation`, we will always want to know the exact one we have.